modules/rosa-sdpolicy-platform.adoc

```diff
@@ -71,65 +71,18 @@ clusters at this time. However, custom labels are supported when creating new ma
 [id="rosa-sdpolicy-backup-policy_{context}"]
 == Cluster backup policy
 
-[IMPORTANT]
-====
-Red{nbsp}Hat does not provide a backup method for
-ifndef::openshift-rosa-hcp[]
-ROSA clusters that use STS.
-endif::openshift-rosa-hcp[]
-ifdef::openshift-rosa-hcp[]
-{hcp-title} clusters.
-endif::openshift-rosa-hcp[]
-It is critical that customers have a backup plan for their applications and application data.
-====
+Red Hat recommends object-level backup solutions for ROSA clusters. OpenShift API for Data Protection (OADP) is included in OpenShift but not enabled by default. Customers can configure OADP on their clusters to achieve object-level backup and restore capabilities.
 
-Application and application data backups are not a part of the
-ifdef::openshift-rosa-hcp[]
-{hcp-title} service.
-endif::openshift-rosa-hcp[]
-ifndef::openshift-rosa-hcp[]
-{product-title} service.
+//Omitted until XCMSTRAT-480 is complete
+//While Red Hat takes frequent backups of etcd, this is for use by Red Hat for maintenance and service restoration purposes, and is never provided to customers for any reason.
 
-ifndef::openshift-rosa-hcp[]
+Red Hat does not back up customer applications or application data. Customers are solely responsible for applications and their data, and must put their own backup and restore capabilities in place.
 
-[%collapsible]
+[WARNING]
 ====
-The table below only applies to non-STS clusters. The following components are used by Red Hat in extenuating circumstances.
-
-//Verify if the corresponding tables in policy-incident.adoc and rosa-policy-incident.adoc also need to be updated.
-
-[cols= "3a,2a,2a,3a",options="header"]
-
-|===
-|Component
-|Snapshot frequency
-|Retention
-|Notes
-
-.2+|Full object store backup
-|Daily
-|7 days
-.2+|This is a full backup of all Kubernetes objects like etcd. No persistent volumes (PVs) are backed up in this backup schedule.
-
-|Weekly
-|30 days
-
-|Full object store backup
-|Hourly
-|24 hour
-|This is a full backup of all Kubernetes objects like etcd. No PVs are backed up in this backup schedule.
-
-|Node root volume
-|Never
-|N/A
-|Nodes are considered to be short-term. Nothing critical should be stored on a node's root volume.
-|===
-
-endif::openshift-rosa-hcp[]
+Customers are solely responsible for backing up and restoring their applications and application data. For more information about customer responsibilities, refer to _Shared responsibility matrix_.
```

Comment:
Per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations) 
```suggestion
Customers are solely responsible for backing up and restoring their applications and application data. For more information about customer responsibilities, see "Shared responsibility matrix".
```